src/components/OrderForm.js

```diff
@@ -305,6 +307,21 @@ class OrderForm extends React.Component {
   componentDidMount() {
     this._isMounted = true;
     this.UNSAFE_componentWillReceiveProps(this.props);
+
+    // Recaptcha
+    getRecaptcha()
```

Comment:
So will this hide the OrderForm until the captcha is completed?

Reply:
No, it does nothing blocking, it's just computing a Recaptcha token that we send to the createOrder mutation on the GraphQL API.